report_thesis/src/sections/proposed_approach/model_selection.tex

```diff
@@ -0,0 +1,103 @@
+Additionally, the models had to be suitable for regression tasks and if no research was available in the context of \gls{libs} data, they had to be models that have shown promise in other domains.
+Our literature review found that a variety of models fit this criteria.
+For example, \citet{andersonPostlandingMajorElement2022} demonstrated that that models such as \gls{gbr}, \gls{pls}, \gls{lasso}, \gls{rf} were each effective at predicting different major oxides.
+Additionaly, \citet{svrforlibs} demonstrated in their work that \gls{svr} outperforms \gls{plsr} in predicting \ce{Si}, \ce{Ca}, \ce{Mg}, \ce{Fe} and \ce{Al}.
+As a result, we have included \gls{gbr}, \gls{pls}, \gls{lasso}, \gls{rf}, and \gls{svr} in our experiments.
+
+In the neural network domain, \cite{ann_libs_soil_analysis} showed that on \gls{libs} data their 3-layer \gls{ann} relative error of prediction for \ce{Ca}, \ce{Fe}, \ce{Al} was below 20\%.
+Likewise, \citet{yangConvolutionalNeuralNetwork2022} showed that \gls{cnn} outperformed methods such as \gls{logreg}, \gls{svm} and linear discriminant analysis at correctly classifying twelve different types of rocks based on \gls{libs} data. 
+While this example for \gls{cnn} is a classification task, \gls{cnn} can be adjusted for regression tasks by changing the loss function and output layer.
+Based on these factors, we decided to include \gls{ann} and \gls{cnn} in our experiments to further increase the diversity of our model selection.
+
+To further bolster our selection pool, we chose to include models that were in the same family of models as those that showed promise in the literature.
+
+\gls{xgboost} was included as an option based on its promising accuracy in various settings.
+For example \citet{xgboost_in_biomedicie} showed that \gls{xgboost} outperfomed models such as \gls{rf} and \gls{svm} in predicting biological activity based on quantitative description of the compound's molecular structure. 
+Another example is \citet{xgboost_in_heart_disease} that used \gls{xgboost} to predict heart disease, where \gls{xgboost} outperformed \gls{rf} and \gls{etr} at correctly classifying patients with heart disease.
+Because of these factors and that \gls{xgboost} has not been widely studied in the context of \gls{libs} data, we decided to include it in our experiments.
+
+Following the same logic, we included \gls{ngboost}. 
+\gls{ngboost} is a recent model introduced by \citet{duan_ngboost_2020} that according to their work improves upon \gls{gbr} by using a more sophisticated loss function and a more advanced gradient boosting algorithm.
+A very limited amount of research has been conducted using this algorithm in the context of \gls{libs} data, however, \citet{ngboost_landslide} showed that \gls{ngboost} outperformed \gls{xgboost} and \gls{rf} and correctly determining landslide-prone fields with an AUC of 0.898 compared to an AUC of 0.871 and 0.863 of \gls{xgboost} and \gls{rf} respectively.
```

Comment:
Need glossary for AUC? Area Under Curve? + then it's classification I think, so not great either. Not sure we need these cites to say that we're exploring these models. That's a good enough reason in itself (exploration).

Reply:
So exclude the numbers and just say it performed better than the other models?

Reply:
My point was actually more about it not being necessary to justify like that. Since we've already clarified some models were included for explorative reasons, it's fine to say this is one of them.